src/Admin/AbstractAdmin.php

```diff
@@ -1570,10 +1586,19 @@ public function setFilterPersister(?FilterPersisterInterface $filterPersister =
     }
 
     /**
+     * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since since sonata-project/sonata-admin-bundle 3.x, to be removed in 4.0.
```

Comment:
@VincentLanglet, could you please fix these messages in a new PR?
```diff
-     * @deprecated since since sonata-project/sonata-admin-bundle 3.x, to be removed in 4.0.
+     * @deprecated since sonata-project/admin-bundle 3.x, to be removed in 4.0.
```